docs/src/introduction/community-contributions.md

```diff
@@ -0,0 +1,20 @@
+# Community Contributions
+
+As an open-source project, Zeebe welcomes contributions and extensions from the community.
+
+We use [Awesome Zeebe](https://awesome.zeebe.io/) as a place to keep track of Zeebe...
+
+* Clients
+* Workers
+* Exporters
+* Applications
+
+...along with other integrations such as Spring-Zeebe and an Apache Kafka connector.
+
+If you contributed something to Zeebe, we encourage you to [add it to Awesome Zeebe via pull request](https://github.com/zeebe-io/awesome-zeebe/blob/master/CONTRIBUTING.md).
```

Comment:
I think the distinction between contributing to Zeebe and contributing to the Zeebe ecosystem is not clear here.

Reply:
Good point, I updated this.